hack/boilerplate.go.txt

```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 The Casbin Authors. All Rights Reserved.
```

Comment:
What's this file used for?

Reply:
There will be codes generated by k8s official tools, and this file is used to stipulate the headers we want to insert in those generated file